packages/d/duckdb/xmake.lua

```diff
@@ -0,0 +1,32 @@
+package("duckdb")
+    set_homepage("http://duckdb.org/")
+    set_description("DuckDB is an in-process SQL OLAP Database Management System")
+    set_license("MIT")
+
+    add_urls("https://github.com/duckdb/duckdb/releases/download/$(version)/libduckdb-src.zip",
+             "https://github.com/duckdb/duckdb.git")
+    add_versions("v0.10.0", "385e27aa67712813e4a07389465c4c5c45c431d97cddd35713b8a306d2a86f2d")
+
+    on_install("macosx", "linux", "windows", function (package)
```

Comment:
is there a reason to restrict it to theses platforms?

Reply:
Because the upstream provide binary for these platforms.